### DISCUSSION TITLE
Dragon Ball Theme

### DISCUSSION THREAD

**Discussion by YasinAl** [mathoudebine/turing-smart-screen-python#156] [Themes]
Here is a Dragon Ball theme that I made (Thanks to @mathoudebine for extracting the background image from the .data file of the application)
![DragonBall_Theme_Example](https://user-images.githubusercontent.com/65172896/217549510-149913ac-ef4e-4f61-8f5e-6d768483a02c.png)

**Updated  for latest release with slight fixes**
[DragonBall.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/10784957/DragonBall.zip)

If you have any suggestions of how I can improve it, let me know!

**Comment by mathoudebine** [comment]
Very nice theme! Thanks for sharing it with the community, you made a great job and it is even better than the original theme from Turing app!

**Reply by YasinAl** [reply]
I will make some improvements to it and update the files. Currently the CPU usage bar is not updating for some reason...
